src/network/connection.cpp

```diff
@@ -903,6 +901,10 @@ void Peer::Drop()
 UDPPeer::UDPPeer(u16 a_id, Address a_address, Connection* connection) :
 	Peer(a_address,a_id,connection)
 {
+	for(unsigned int i=0; i< CHANNEL_COUNT; i++)
```

Comment:
`for (size_t i = 0; i < CHANNEL_COUNT` or use the C++11 range-based "for" loop.
The brackets can be omitted, otherwise please correct the placement.